### PR TITLE
upgrade ruby to 2.3.0, rails to 4.2.5.x

### PR DESCRIPTION
**Pull request by dunn**
Initially bumped the Ruby version to deal with https://github.com/rbenv/ruby-build/issues/526 but 2.1 was a while ago so this might be worthwhile in general.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ gem 'jbuilder', '~> 1.2'
 gem 'piwik_analytics', '~> 1.0.2'
 
 gem 'geocoder', '~> 1.2.1'
-gem 'activeadmin', git: 'https://github.com/gregbell/active_admin.git'
+gem 'activeadmin', '1.0.0.pre2'
```
Comment (dunn) [inline Gemfile]: This isn't strictly necessary, but there's a quirk in that repository that causes fetches to fail when the local Git settings enforce strict integrity checking: https://github.com/activeadmin/activeadmin/issues/4361

**Comment by dunn** [comment]
The Travis failure (https://travis-ci.org/RefugeRestrooms/refugerestrooms/builds/118330840#L1485) should be fixed by #257

**Comment by tkwidmer** [comment]
Can you merge develop, which should have #257 now?

**Comment by dunn** [comment]
OK, rebased!

Let me know if you'd like this split up into multiple commits, too.

**Comment by tkwidmer** [comment]
lgtm!

**Comment by dunn** [comment]
Thanks!

**Pull request merged**